### ISSUE TITLE
python2.4 compat issue: must be type, not classobj

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#155]
When the documentation is built with CentOS 5.3 (having python2.4), the following error is given:

```
#!python

Traceback (most recent call last):
  File "/usr/lib/python2.4/site-packages/Sphinx-1.0dev_20090505-py2.4.egg/sphinx/cmdline.py", line 171, in main
    warningiserror, tags)
  File "/usr/lib/python2.4/site-packages/Sphinx-1.0dev_20090505-py2.4.egg/sphinx/application.py", line 97, in __init__
    self.setup_extension(extension)
  File "/usr/lib/python2.4/site-packages/Sphinx-1.0dev_20090505-py2.4.egg/sphinx/application.py", line 169, in setup_extension
    err)
  File "/usr/lib/python2.4/site-packages/Sphinx-1.0dev_20090505-py2.4.egg/sphinx/errors.py", line 31, in __init__
    super(ExtensionError, self).__init__(message)
TypeError: super() argument 1 must be type, not classobj
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/155
- Originally reported by: Anonymous
- Originally created at: 2009-05-05T13:03:00.446

**Comment by shimizukawa** [comment]
_From Anonymous on 2009-05-05 12:16:59+00:00_

Actually, the reason for the error //was not python2.4// but **non-existing extension module**:

If the there is non-existing module defined in **extensions** (in the conf.py), the error shown above is shown. Fortunatelly the //ExtensionError// gave a hint were to look from.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2009-05-10 19:14:55+00:00_

Fixed in <<cset b1dbe952552cd265c4e554f0068a47a1afca8b69>>, thanks!
